web/src/test/java/org/openmrs/web/filter/update/UpdateFilterModelTest.java

```diff
@@ -126,36 +136,36 @@ public void createUpdateFilterModel_shouldNotRequireAnUpdateIfChangesAreNullAndD
 	public void createUpdateFilterModel_shouldNotRequireAnUpdateIfDatabaseUpdaterIsLockedAndCallingDatabaseUpdaterTwiceAlwaysReturnsNull()
 	        throws Exception {
 		
-		when(DatabaseUpdater.getUnrunDatabaseChanges()).thenReturn(null);
+		when(DatabaseUpdater.getUnrunDatabaseChanges(any(LiquibaseProvider.class))).thenReturn(null);
 		when(DatabaseUpdater.isLocked()).thenReturn(true);
 		when(DatabaseUpdater.updatesRequired()).thenReturn(false);
 		
-		model = new UpdateFilterModel();
+		model = new UpdateFilterModel(liquibaseProvider);
 		
 		assertFalse("should not require an update", model.updateRequired);
 		assertNull("should not have changes", model.changes);
+		
 		PowerMockito.verifyStatic(DatabaseUpdater.class, times(2));
-		DatabaseUpdater.getUnrunDatabaseChanges();
+		DatabaseUpdater.getUnrunDatabaseChanges(liquibaseProvider);
 		PowerMockito.verifyStatic(DatabaseUpdater.class);
 		DatabaseUpdater.updatesRequired();
 	}
 	
 	@Test
 	public void createUpdateFilterModel_shouldRequireAnUpdateIfDatabaseUpdaterIsLockedAndChangesAreNotNull()
 	        throws Exception {
+		List<OpenMRSChangeSet> changes = Arrays.asList(mock(OpenMRSChangeSet.class));
 		
-		OpenMRSChangeSet change = mock(OpenMRSChangeSet.class);
-		List<OpenMRSChangeSet> changes = new ArrayList<>();
-		changes.add(change);
-		when(DatabaseUpdater.getUnrunDatabaseChanges()).thenReturn(null, changes);
+		when(DatabaseUpdater.getUnrunDatabaseChanges(any(LiquibaseProvider.class))).thenReturn(changes);
 		when(DatabaseUpdater.isLocked()).thenReturn(true);
 		
-		model = new UpdateFilterModel();
+		model = new UpdateFilterModel(liquibaseProvider);
 		
 		assertTrue("should require an update", model.updateRequired);
 		assertThat(model.changes, is(changes));
-		PowerMockito.verifyStatic(DatabaseUpdater.class, times(2));
```

Comment:
Did you intentionally remove `times(2)`?

Reply:
Yes, the design of that class is somewhat convoluted, I padded it in tests but decided not to change it too much.